### PR TITLE
change relative paths

### PR DESCRIPTION
**Pull request by Daniel162-hub**

### CODE REVIEW

File: src/tests/MCTS/MCTSTests.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import unittest
 
-from src.mcts.MCTSSolver import MCTSSolver, max_iter, max_time_s, expansion_choose_meth
-from src.tests.MCTS.NaughtCrossesState import NaughtCrossesState
+from mcts.MCTSSolver import MCTSSolver, max_iter, max_time_s, expansion_choose_meth
```
Comment (steviej08) [inline src/tests/MCTS/MCTSTests.py:3]: What, this is terrible, why have you done this. Change it!!

Comment (Daniel162-hub) [reply src/tests/MCTS/MCTSTests.py:3]: no

**Review by steviej08 — CHANGES_REQUESTED** [review]
All the things

**Review by steviej08 — APPROVED** [review]
Okay actually looks fine

**Pull request merged**